src/movies.js

```diff
@@ -1,16 +1,89 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(){
+    const listOfDirectors = movies.map(movies => movies.director) 
+    return listOfDirectors
+};
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
 
+const howManyMovies = listOfMovies => {
+    let stevenMovies = listOfMovies.filter(list => list['director'] === 'Steven Spielberg');  
+    let solution = stevenMovies.filter (dramas => dramas['genre'].includes('Drama')); 
+    return solution.length;}
+
```

Comment:
Good work Roy!

Here's an alternative way to write function howManyMovies

`function howManyMovies(someMovies) {

return someMovies.filter(
eachMovie =>
eachMovie.director === 'Steven Spielberg' &&
eachMovie.genre.includes('Drama')
).length;
}`